packages/core/src/anchor/anchor-processing-loop.ts

```diff
@@ -111,4 +124,18 @@ export class AnchorProcessingLoop {
     this.#anchorPollingMetrics.stopPublishingStats()
     return this.#loop.stop()
   }
+
+  /**
+   * Check the poll time for a stream. Updates the poll time and returns true if the stream has not been polled in the
+   * last pacing interval.
+   */
+  checkPollTime(streamId: string): boolean {
+    if (this.#pollCache.has(streamId) && (Date.now() < this.#pollCache.get(streamId))) {
```

Comment:
nitpick but would be nice to only call into the cache once and just check for null vs having a separate lookup for the `has` method.  Hashtable lookups do have some performance cost due to the hashing, so it's nice to minimize how often you need to go into them.

Reply:
Something like this?
```
    const nextPollTime = this.#pollCache.get(streamId)
    if (nextPollTime && (Date.now() < nextPollTime)) {
      return false
    }

```

Reply:
yeah exactly.